### PR TITLE
Alluxio Enterprise Edition - DC/OS Package - v0

### PR DESCRIPTION
**Pull request by madanadit**
Initial version of the universe package for installing Alluxio Enterprise Edition.

### CODE REVIEW

File: repo/packages/A/alluxio-enterprise/0/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "packagingVersion": "3.0",
+  "name": "alluxio-enterprise",
+  "version": "1.4.0",
+  "maintainer": "info@alluxio.com",
+  "description": "Alluxio Enterprise Edition (AEE). Please obtain a license by visiting: https://alluxio.com/products \n\n Alluxio, formerly Tachyon, is the world's first system that unifies disparate storage systems at memory speed. With Alluxio, enterprises can manage data efficiently, accelerate business analytics, and ease the adoption of hybrid cloud. \n\n Please refer to https://alluxio.com/docs/ for a detailed operation guide.",
```
Comment (ryadav88) [inline repo/packages/A/alluxio-enterprise/0/package.json]: do we have release notes?

File: repo/packages/A/alluxio-enterprise/0/resource.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+  },
+  "images": {
+    "icon-small": "https://s3-us-west-1.amazonaws.com/alluxio-dcos-assets/icons/alluxio_logo_small.png?raw=true",
+    "icon-medium": "https://s3-us-west-1.amazonaws.com/alluxio-dcos-assets/icons/alluxio_logo_medium.png?raw=true",
+    "icon-large": "https://s3-us-west-1.amazonaws.com/alluxio-dcos-assets/icons/alluxio_logo_large.png?raw=true"
+  },
+  "cli":{
+    "binaries":{
+      "darwin":{
```
Comment (ryadav88) [inline repo/packages/A/alluxio-enterprise/0/resource.json]: `dcos package install alluxio-enterprise --cli` gives `The hash for the downloaded subcommand [15a9e457e995337b25799a70cd6d93b1133461d722959d264f19053e2b017ba3] does not match the expected value [{{sha256:dcos-alluxio-enterprise-darwin}}]. Aborting...`

File: repo/packages/A/alluxio-enterprise/0/package.json (hunk 1 above)
Comment (madanadit) [inline repo/packages/A/alluxio-enterprise/0/package.json]: @ryadav88 the install guide will be available at alluxio.com/docs (already linked) once i push that live

Comment (ryadav88) [inline repo/packages/A/alluxio-enterprise/0/package.json]: Can we add a direct link to the dc/os page? https://alluxio.com/docs/enterprise/1.4/en/Alluxio-on-DCOS.html ? is this current

Comment (madanadit) [inline repo/packages/A/alluxio-enterprise/0/package.json]: Sure, It's not current but the link is going to be the same.

**Pull request merged**